README.md

```diff
@@ -1,12 +1,28 @@
-# Agenda
-[![Build Status](https://api.travis-ci.org/agenda/agenda.svg)](http://travis-ci.org/agenda/agenda)
-[![dependencies Status](https://david-dm.org/agenda/agenda/status.svg)](https://david-dm.org/agenda/agenda)
-[![devDependencies Status](https://david-dm.org/agenda/agenda/dev-status.svg)](https://david-dm.org/agenda/agenda?type=dev)
-[![Coverage Status](https://coveralls.io/repos/github/agenda/agenda/badge.svg?branch=master)](https://coveralls.io/github/agenda/agenda?branch=master)
-
-Agenda is a light-weight job scheduling library for Node.js.
-
-It offers:
+<p align="center">
+  <img src="https://cdn.rawgit.com/agenda/agenda/update-readme-header/agenda.svg" alt="Agenda" width="100" height="100">
```

Comment:
Not sure if you were planning on doing this already, but I'd replace `update-readme-header` with `master` before merging, just in case this branch ever gets pruned 😅